packages/grafana-ui/src/components/Table/ImageCell.tsx

```diff
@@ -15,13 +15,13 @@ export const ImageCell: FC<TableCellProps> = (props) => {
 
   return (
     <div {...cellProps} className={tableStyles.cellContainer}>
-      {!hasLinks && <img src={displayValue.text} className={tableStyles.imageCell} />}
+      {!hasLinks && <img src={displayValue.text} className={tableStyles.imageCell} alt="" />}
```

Comment:
This all looks great, nice job 🙌  Just one thing that I was thinking while reading this. In some of these ones that we have empty alts because the images are passed in (like here or `SelectOption` for example), do we want to add a prop for `imgAlt`? This would let whoever uses this component pass the alt themselves if they find it useful and if not we just default it to empty string. I don't know if this is even possible/easy to do. What do you think? Could also create separate issues to do this.

Reply:
Good idea @JoaoSilvaGrafana. Just not sure whether to create a separate issue or not. What do you think about adding a prop for `imgAlt` @ashharrison90?

Reply:
`SelectOption` i think we could argue is decorative because you will always have the option text there. 

for `ImageCell` i think that it would be nice, but i don't think there's any mechanism for us to do that currently with the table component since all it's doing is taking the `displayValue` from the data and then using that as a src for the image. i'm not sure how/where in the data we'd put the `imgAlt`. 🤔 

either way, feels like a fairly meaty task which is probably beyond the scope of this PR.

Reply:
Yep agree with this, the PR does the original task this is an extra thing 🙏